### DISCUSSION TITLE
localizing forms

### DISCUSSION THREAD

**Discussion by fredbouin** [statamic/cms#4970] [Troubleshooting]
How does one go about localizing a form? From what I'm seeing so far it doesn't look like there is a way to change the blueprint labels and arrays. Should I just create another form entirely for the second language or is there a more elegant solution I'm missing?

Update: I tried going with an entirely new form but things like "please select" in the select menus are still in english. If I go with the two separate form option how do I go about making my form entirely in whatever language I want it to be.

**Comment by edalzell** [comment]
Go give this [FR](https://github.com/statamic/ideas/issues/50) a thumbs up
